include/sys/libc-hooks.h

```diff
@@ -67,10 +67,15 @@ extern struct k_mem_partition z_malloc_partition;
 extern struct k_mem_partition z_malloc_partition;
 #endif
 
-#if defined(CONFIG_NEWLIB_LIBC) || defined(CONFIG_STACK_CANARIES) || \
-    defined(CONFIG_NEED_LIBC_MEM_PARTITION)
-/* Minimal libc has no globals. We do put the stack canary global in the
- * libc partition since it is not worth placing in a partition of its own.
+#if defined(CONFIG_MINIMAL_LIBC_GLOBALS) || defined(CONFIG_NEWLIB_LIBC) || \
+	defined(CONFIG_STACK_CANARIES) || defined(CONFIG_NEED_LIBC_MEM_PARTITION)
```

Comment:
`CONFIG_MINIMAL_LIBC_GLOBALS` should not be necessary if we use `CONFIG_NEED_LIBC_MEM_PARTITION` as recommended above.

Reply:
done